src/main/resources/google_checks.xml

```diff
@@ -73,7 +73,7 @@
             <property name="option" value="alone"/>
             <property name="tokens"
              value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT,
-                    INSTANCE_INIT"/>
+                    INSTANCE_INIT, ANNOTATION_DEF"/>
```

Comment:
RightCurly is located at for google_style: https://checkstyle.sourceforge.io/google_style.html#a4.1.2
https://checkstyle.sourceforge.io/styleguides/google-java-style-20180523/javaguide.html#s4.1.2-blocks-k-r-style

I don't see any issues with it following the same pattern as CLASS_DEF.
Please add an example input to RightCurly under the IT folder, specifically the google regression.